ports/esp8266/boards/GENERIC_512K/manifest.py

```diff
@@ -1,8 +1,9 @@
 module("_boot.py", opt=3)
 module("apa102.py", base_path="$(PORT_DIR)/modules", opt=3)
 module("port_diag.py", base_path="$(PORT_DIR)/modules", opt=3)
-module("ntptime.py", base_path="$(MPY_DIR)/extmod", opt=3)
-include("$(MPY_DIR)/drivers/dht")
-include("$(MPY_DIR)/drivers/onewire")
-include("$(MPY_DIR)/extmod/webrepl")
-include("$(MPY_DIR)/drivers/neopixel")
+require("ntptime.py")
```

Comment:
should this be `"ntptime"`?  does it work if you have the `.py`?  does it give an error or silently fail?

Reply:
Fixed.